gen/templates/generic/data_source.go

```diff
@@ -58,10 +58,12 @@ func (d *{{camelCase .Name}}DataSource) Schema(ctx context.Context, req datasour
 		MarkdownDescription: "{{.DsDescription}}",
 
 		Attributes: map[string]schema.Attribute{
+			{{- if not .RemoveId }}
```

Comment:
I think we should not necessarily remove the id attribute from the schema, we should just populate it with something else (chassis_id, name, etc.).

Reply:
I was debating that. If anything I was thinking it might make sense especially for a querying endpoint to be a collection of its inputs so thats an identifier to what its doing. Thoughts there? Cause when fetching an array, there is no identifier